### DISCUSSION TITLE
rAudio 20230101 - no backup restore

### DISCUSSION THREAD

**Discussion by kw-sudo** [rern/rAudio#1183] [Fixed Issues]
Hi,

after creating a new sd-card (basic full image i20221117 + update), I wanted to restore my 'old' settings (also 20230101). r/Audio is stell keeping default settings, no changes.

RPi3B+

Regards,
Oliver

**Comment by hjfisch** [comment]
Same for me . I needed to do a new config , that causes some time , but now it is working without probs for me . So create a new card without restoring your settings ( will cause some other probs too).

**Reply by kw-sudo** [reply]
So I did.  I don't know what went wrong with all the partial updates in the past, but I did a clean install from scratch (i20221117 + addons) and full system upgrade (_pacman -Syu_) and in a wondrous way everything's works fine now.

**Comment by digixmax** [comment]
FWIW I too was not able to perform a restore after a clean flash of 20221117 plus update to 20230101 -- the restore operation appeared successfully completed but rAudio configuration settings remained unchanged at their default values following reboot!

**Comment by newbee90** [comment]
you can do it manualy, unzip or unrar your backup.gz und backup file inside backup.gz
database is /srv/http/data/mpd
folder config goes to root
i use winscp

**Reply by kw-sudo** [reply]
Thank you, but I could 'fix' it with a clean install from scratch. This time it works. ;-)
